glib-macros/src/object_impl_attributes.rs

```diff
@@ -0,0 +1,134 @@
+pub mod interface;
```

Comment:
Needs the file header like all the other files here (that's why the CI is unhappy)